From 79b16f82546fc14f7ea68bcaff822e946a376aa2 Mon Sep 17 00:00:00 2001
From: QiuJi <qiuji@iscas.ac.cn>
Date: Wed, 16 Jun 2021 11:05:53 +0800
Subject: [PATCH] [riscv64] Add riscv64 to the architecture list of
 js-perf-test

Change-Id: I541973c5b0570c1a1c23ce8e09cd20d3904df749
Reviewed-on: https://chromium-review.googlesource.com/c/v8/v8/+/2966198
Reviewed-by: Michael Achenbach <machenbach@chromium.org>
Commit-Queue: Ji Qiu <qiuji@iscas.ac.cn>
Cr-Commit-Position: refs/heads/master@{#75210}
---
 tools/run_perf.py | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/tools/run_perf.py b/tools/run_perf.py
index 80ea1f95659..cdbbed8176f 100644
--- a/tools/run_perf.py
+++ b/tools/run_perf.py
@@ -136,7 +136,8 @@ SUPPORTED_ARCHS = ['arm',
                    'mips',
                    'mipsel',
                    'x64',
-                   'arm64']
+                   'arm64',
+                   'riscv64']
 
 GENERIC_RESULTS_RE = re.compile(r'^RESULT ([^:]+): ([^=]+)= ([^ ]+) ([^ ]*)$')
 RESULT_STDDEV_RE = re.compile(r'^\{([^\}]+)\}$')
-- 
2.35.1

